paths/job_annotations/destroy.yaml

```
@@ -0,0 +1,46 @@
+---
+summary: Delete a job annotation
+description: Delete an annotation for a job.
+operationId: job_annotation/destroy
```

we usually call the operation `delete`,

eg

https://github.com/phrase/openapi/blob/main/paths/branches/destroy.yaml#L4

```suggestion
operationId: job_annotation/delete
```